controller/contacts.js

```diff
@@ -0,0 +1,50 @@
+const {HttpError, controlWrapper} = require('../helpers');
+const contacts = require('../models/contacts');
```

Comment:
```suggestion
const contacts = require('../models/contacts');
```
можна зробити деструктуризацію